com.longbridgeapp.LongbridgePro.yaml

```diff
@@ -0,0 +1,78 @@
+app-id: com.longbridgeapp.LongbridgePro
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: longbridgepro
+
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
Can you check if notifications work without this, locally? `flatpak run --no-talk-name=org.freedesktop.Notifications com.longbridgeapp.LongbridgePro` 

The 23.08 electron baseapp has libnotify 0.8.2 which is supposed to not require this sandbox hole.